index.js

```diff
@@ -48,7 +52,12 @@ const toJSON = from => {
 	return json;
 };
 
-const getErrorConstructor = name => errorConstructors.get(name) ?? Error;
+const newError = name => {
+	const ErrorConstructor = errorConstructors.get(name) ?? Error;
+	return ErrorConstructor === globalThis.AggregateError
```

Comment:
Node 15+
Safari 14+

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError#browser_compatibility


Maybe it can just be 
```suggestion
	return ErrorConstructor === AggregateError
```